### ISSUE TITLE
cant start server

### ISSUE THREAD

**Issue by CalvinDKL** [Megabyte918/MultiOgar-Edited#576]
<img width="723" alt="screen shot 2017-02-21 at 09 27 19" src="https://cloud.githubusercontent.com/assets/23283642/23162975/db327476-f830-11e6-8a50-75a2fd12ca65.png">

Anyone know how to fix?

**Comment by MartincoderMlg** [comment]
npm install

**Comment by Megabyte918** [comment]
@CalvinDKL Can you show the full error? It would help

**Comment by CalvinDKL** [comment]
that is the full error

**Comment by CalvinDKL** [comment]
<img width="723" alt="screen shot 2017-02-21 at 09 27 19" src="https://cloud.githubusercontent.com/assets/23283642/23156600/f74c9f9c-f817-11e6-8f65-e96fcc48a37a.png">
Here is another image, where i ran it normally instead of using pm2 and logging it.

**Comment by AlexHGaming** [comment]
`1. Delete your node_modules folder.`
`2. Go to package.json, and change from "ws": "latest" to "ws": "1.1.0"`
`3. Do the npm install again`
Thank me later :)

**Comment by CalvinDKL** [comment]
hahahahaha, let me try.

**Comment by CalvinDKL** [comment]
Nope, didn't work.

**Comment by CalvinDKL** [comment]
Same error

**Comment by ItzLevvie** [comment]
@CalvinDKL https://github.com/JaraLowell/OgarServ/issues/172#issuecomment-279242778

**Comment by Andrews54757** [comment]
@CalvinDKL Regarding to #603. The problem is that


1. The url field is not a url, it is a string. Your server then will be accessable via `opnagar.us/server/[string]`. Fill in the `[string]` with what you put there.
2. Your port is not open. You need to port foward within your router.

**Comment by MastaCoder** [comment]
here goes another andrew rant LOOOOOOL

**Comment by MastaCoder** [comment]
@Andrews54757 I don't like you, I really don't like you. However, the points you made make sense.

As I said before, and I'm going to say again. You're putting in too much work for too little of a fanbase. You have 92 people with OpenAgar, and I'm pretty sure not even 1/10 of them know anything about Node.js. (Based on some of the issues I saw before you decided to close it.)

Sure I'll take over, send me the actual clean files of the server, it's really not that hard to run a project with questions like this. I did it with Cigar a few months back and it worked fine. Had 10 issues open, all relevant to the project. (I stopped Cigar dev for a few months while Luka and I can work on abs0rb's new update) How will I change it? Here's how:

1. If someone asks a question about modifying code, answer we can't help with that and close it.
2. Same with useless questions or duplicates. I wouldn't even try to tag the duplicate, I'll let them to go search for it. You'll never learn if you're spoon-fed.
3. Cut down costs. Use more subdomains and use free domains. Contributions don't need to be too advanced, just vanilla agar.io. End of story. Servers? I won't even contribute to servers, you self host it or don't use the project at all.

You're making something for a small group of people, while projects like MultiOgar-Edited are made for a large group of people that want a self-hosted project. If you're looking to expand, this is not even close to the way to expand. It's really easy to cleanup bot code and check the backend if the servers are available for use.

MultiOgar-Edited is doing fine. Many of the features are fine and don't seem to affect performance that much. (However, console-plus and that mobile split feature, I find them completely useless lol) 

TL;DR: You're putting in too much work Andrew. 

CY@@@@@@@@@@ FAMS!
